### ISSUE TITLE
Problem building html file. 

### ISSUE THREAD

**Issue by rmbzz** [arechnitzer/CLP1#36]
I have just downloaded (cloned) the source for CLP1 and ran into a small problem with the instructions in the file pretext/readme.md

This file suggests I use the command line 
`pretext build html -g all`
to build the html version of the site. However, this seems to fail with the error message:
 
> Error: Got unexpected extra argument (all)

When I ran the build command without the option -g all. That is, I used
`pretext build html`
It appears that this command finished without an error and appeared to build a usable copy of the text. 

I am using PreTeXt-CLI version: 2.6.1 which was installed using the suggested command
`pip3 install pretext`
on an installation of Ubuntu 22.04. 

Is there likely to be some non obvious problem if I build without the -g all option? If so, how would I check? 

I also had the same problem with CLP2. I have not gotten to 3 and 4 yet and I have not tried to build the pdf files yet. 

Thanks
Russell Brown

**Comment by arechnitzer** [comment]
Looking into this presently = we are doing our annual "clean up CLP issues" stuff.

**Comment by arechnitzer** [comment]
~~If you run "pretext build html -g" do things work?~~

ops - sorry - I should read more carefully. Can you also build the latex from pretext sources?

**Comment by rmbzz** [comment]
1. pretext build html -g seems to work as well as pretext build html.  Which is the correct command? 
2. The first problem is that I am not sure what the correct command to build LaTeX. 
I tried 
`pretext build pdf`
which failed with the errors 

> critical: [Errno 2] No such file or directory: 'clp_1_dc.pdf'
> critical: Failed to build.  Exiting...

and 
`pretext build latex` 
also failed to build. The error message is not easily readable. 

In the past, I have not tried to build the LaTeX version of the book using pretext. Instead I use the latex source that is distributed to create a pdf version from LaTeX. So the failure of the LaTeX build is not an issue for me. 


Since my first message, I have tried to build the problem book for CLP1 using pdflatex and the provided LaTeX source.   This seems to be successful. 

Russell Brown
